api/payments.go

```diff
@@ -167,6 +167,14 @@ func (a *API) PaymentCreate(w http.ResponseWriter, r *http.Request) error {
 	tr.ProcessorID = processorID
 	tr.InvoiceNumber = invoiceNumber
 
+	if pendingErr, ok := err.(*payments.PaymentPendingError); ok {
```

Comment:
doesn't this need to be under an `if err != nil` block? otherwise you're checking if nil could be that?